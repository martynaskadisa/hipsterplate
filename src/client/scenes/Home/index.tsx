import * as React from 'react';
import { RouteComponentProps } from 'react-router-dom';

const Home: React.SFC<RouteComponentProps<any>> = () => {
  return <div>Home</div>;
};

export default Home;
