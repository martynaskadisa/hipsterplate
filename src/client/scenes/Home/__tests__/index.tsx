import * as React from 'react';
import * as renderer from 'react-test-renderer';

import Home from '../index';

describe('Home scene', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Home />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
