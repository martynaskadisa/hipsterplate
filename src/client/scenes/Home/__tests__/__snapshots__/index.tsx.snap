// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home scene should render correctly 1`] = `
<div>
  Home
</div>
`;
